samples/glean/src/main/java/org/mozilla/samples/glean/MainActivity.kt

```diff
@@ -13,12 +14,20 @@ import org.mozilla.samples.glean.GleanMetrics.Test
 import org.mozilla.samples.glean.GleanMetrics.BrowserEngagement
 import org.mozilla.samples.glean.library.SamplesGleanLibrary
 
-open class MainActivity : AppCompatActivity() {
+open class MainActivity : AppCompatActivity(), ExperimentUpdateReceiver.ExperimentUpdateListener {
+
+    private var experimentUpdateReceiver: ExperimentUpdateReceiver? = null
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
+        // Set up the ExperimentUpdateReceiver to receive experiment updated Intents
```

Comment:
Can you add a comment stating that this is not relevant to the Glean SDK but to the experiments library?